### PR TITLE
Phaser renderer remove pixi chat

### PR DESCRIPTION
**Pull request by sloont**

### CODE REVIEW

File: ts/src/renderer/phaser/classes/PhaserChatBubble.ts [hunk 1]
```diff
@@ -1,55 +1,40 @@
 class PhaserChatBubble extends Phaser.GameObjects.Container {
 
 	private readonly triangle: Phaser.GameObjects.Graphics;
-	private readonly bubble: Phaser.GameObjects.Graphics;
-	private readonly text: Phaser.GameObjects.Text;
+	protected bubble: Phaser.GameObjects.Graphics;
+	protected textObject: Phaser.GameObjects.Text;
```
Comment (sloont) [inline ts/src/renderer/phaser/classes/PhaserChatBubble.ts:5]: test whether this change has an effect. I don't think it did as we no longer adjust textObject `alpha`

File: ts/src/renderer/phaser/classes/PhaserChatBubble.ts [hunk 2]
```diff
@@ -65,50 +50,25 @@ class PhaserChatBubble extends Phaser.GameObjects.Container {
...
+		this.createBubble();
+		
+		this.alpha = 1;
 		this.resetFadeOut ();
+
 		this.fadeOut();
 	}
 
-	private fadeOut(): void {
+	fadeOut(): void {
```
Comment (sloont) [inline ts/src/renderer/phaser/classes/PhaserChatBubble.ts:71]: test whether these can be private again.

File: ts/src/renderer/phaser/classes/PhaserChatBubble.ts [hunk 3]
```diff
@@ -146,6 +105,42 @@ class PhaserChatBubble extends Phaser.GameObjects.Container {
...
+			width * 10 / 20,
+			height,
+			borderRadius
+		);
+
+		bubble.lineStyle(2, 0x000000, 1);
+		this.bubble.x = this.textObject.x + width / 4;
+
+		bubble.setDepth(0);
+		this.add(bubble);
```
Comment (sloont) [inline ts/src/renderer/phaser/classes/PhaserChatBubble.ts:138]: can remove cause we never remove the bubble. this was part of the issue with alpha stacking/faded text

File: ts/src/renderer/phaser/scenes/GameScene.ts [hunk 4]
```diff
@@ -9,7 +9,7 @@ class GameScene extends PhaserScene {
 		// phaser canvas adjustments
 		const canvas = this.game.canvas;
 		canvas.style.position = 'fixed';
-		canvas.style.opacity = '0.5';
+		canvas.style.opacity = '1';
```
Comment (sloont) [inline ts/src/renderer/phaser/scenes/GameScene.ts:12]: revert after tests and before merge to `phaser-renderer`

File: ts/src/renderer/phaser/classes/PhaserChatBubble.ts (hunk 3 above)
Comment (sloont) [reply ts/src/renderer/phaser/classes/PhaserChatBubble.ts:138]: rather: move to constructor

**Review by pavle-goloskokovic — CHANGES_REQUESTED** [review]

**Review by pavle-goloskokovic — APPROVED** [review]

**Pull request merged**